www/js/lib/archive.js

```diff
@@ -333,22 +333,16 @@ define(['normalize_string', 'geometry', 'title', 'util', 'titleIterators', 'q'],
     };
     
     /**
-     * @callback callbackTitle
-     * @param {Title} title Title found
-     */
-
-    /**
      * Look for a title by its name, and call the callbackFunction with this Title
      * If the title is not found, the callbackFunction is called with parameter null
      * @param {String} titleName
-     * @param {callbackTitle} callbackFunction
```

Comment:
If possible add a JSDoc annotation to indicate it now returns a Promise, and modify the description to reflect this change